app/models/course.rb

```diff
@@ -123,6 +123,14 @@ def not_available?
     !exposed_in_find
   end
 
+  def fee_paying?
+    funding_type == 'fee'
+  end
+
+  def salaried?
+    funding_type == 'salary'
+  end
+
```

Comment:
do we also need to think about `apprenticeship`?

Reply:
@MylesJarvis I assume that's ok as apprenticeship is also a type of salaried course?

Reply:
yep, I think best to cover all bases.